snap/local/exporter-wrapper.bash

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+
+PROPERTIES=$(cat "${SNAP_COMMON}"/exporter.properties)
+BOOTSTRAP_SERVERS=$(echo "${PROPERTIES}" | grep bootstrap.servers | sed 's/bootstrap\.servers=//g')
+SECURITY_PROTOCOL=$(echo "${PROPERTIES}" | grep security.protocol | sed 's/security\.protocol=//g')
+SASL_JAAS_CONFIG=$(echo $PROPERTIES | grep sasl.jaas.config | sed 's/sasl\.jaas\.config=//g')
+PASSWORD=$(echo $SASL_JAAS_CONFIG | grep -oP "password\=\"([a-zA-Z0-9]+)" | cut -d "\"" -f 2)
+
+for i in $(echo $BOOTSTRAP_SERVERS | tr "," "\n")
+do
+    args+=("--kafka.server=${i}")
```

Comment:
**question** I'm just wondering this, and probably it would be more of an hassle to handle this than the actual benefit, but I'm wondering whether it would make sense to bind the kafka-exporter only to the server of its snap

It is really not a question for this PR, since anyway it depends how we value the exporter.properties file, but it is something I started to wonder and I thought it was worth bringing it up

Reply:
Actually you may have a point here! I haven't had the chance to check behavior of having N exporters all grabbing from N units. Will validate once I can check in the charm!